### PR TITLE
add jhiMin max

### LINKED ISSUE #7514: Max and Min validators for number input fields  don't work in Angular5
##### **Overview of the issue**

Min and Max validators have no effects on numeric input fields in Angular5.
No error is generated inserting bad values.

##### **Reproduce the error**

Create an entity with a numeric field and insert a max and min constraints.

##### **JHipster Version(s)**

```
test-validation@0.0.0 /Users/sbertini/Temp/_Delete/30/TestValidation
└── generator-jhipster@4.14.3

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.mycompany.myapp"
    },
    "jhipsterVersion": "4.14.3",
    "baseName": "TestValidation",
    "packageName": "com.mycompany.myapp",
    "packageFolder": "com/mycompany/myapp",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "no",
    "websocket": false,
    "databaseType": "sql",
    "devDatabaseType": "h2Disk",
    "prodDatabaseType": "postgresql",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSocialSignIn": false,
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "replaced-by-jhipster-info",
    "clientFramework": "angularX",
    "useSass": false,
    "clientPackageManager": "yarn",
    "applicationType": "monolith",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "enableTranslation": false
  }
}
</pre>
</details>

##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>
entity TestEntity (test_entity) {
  testValue Integer required min(0) max(10)
}

</pre>
</details>

##### **Environment and Tools**

java version "1.8.0_121"
Java(TM) SE Runtime Environment (build 1.8.0_121-b13)
Java HotSpot(TM) 64-Bit Server VM (build 25.121-b13, mixed mode)

git version 2.14.3 (Apple Git-98)

node: v6.10.1

npm: 5.6.0

bower: 1.8.2

gulp:
[18:35:17] CLI version 1.2.2

yeoman: 2.0.0

yarn: 1.3.2

Docker version 17.12.0-ce, build c97c6d6

docker-compose version 1.18.0, build 8dd22a9

- [X] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by Dufgui**
fix #7514

- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs [hunk 1]
```diff
@@ -36,9 +36,11 @@ if (field.fieldValidate === true) {
     }
     if (rules.includes('min')) {
         validators.push('min="' + field.fieldValidateRulesMin + '"');
+        validators.push('jhimin="' + field.fieldValidateRulesMin + '"');
     }
     if (rules.includes('max')) {
         validators.push('max="' + field.fieldValidateRulesMax + '"');
+        validators.push('jhimax="' + field.fieldValidateRulesMax + '"');
```
Comment (ruddell) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs]: jhimax -> `jhiMax`

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs [hunk 2]
```diff
@@ -36,9 +36,11 @@ if (field.fieldValidate === true) {
     }
     if (rules.includes('min')) {
         validators.push('min="' + field.fieldValidateRulesMin + '"');
+        validators.push('jhimin="' + field.fieldValidateRulesMin + '"');
```
Comment (ruddell) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs]: jhimin -> `jhiMin`

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs [hunk 3]
```diff
@@ -36,9 +36,11 @@ if (field.fieldValidate === true) {
     }
     if (rules.includes('min')) {
         validators.push('min="' + field.fieldValidateRulesMin + '"');
```
Comment (ruddell) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs]: Remove this, it doesn't do anything

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs [hunk 4]
```diff
@@ -36,9 +36,11 @@ if (field.fieldValidate === true) {
...
+        validators.push('jhimin="' + field.fieldValidateRulesMin + '"');
     }
     if (rules.includes('max')) {
         validators.push('max="' + field.fieldValidateRulesMax + '"');
```
Comment (ruddell) [inline generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs]: Remove this, it doesn't do anything

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs (hunk 3 above)
Comment (Dufgui) [reply generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs]: no it's use by some browser. Chrome add a slicer limited by min max, try it here : https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_max_min

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs (hunk 4 above)
Comment (Dufgui) [reply generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs]: no it's use by some browser. Chrome add a slicer limited by min max, try it here : https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_max_min

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs (hunk 1 above)
Comment (Dufgui) [reply generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs]: done

File: generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs (hunk 2 above)
Comment (Dufgui) [reply generators/entity-client/templates/angular/src/main/webapp/app/entities/ng_validators.ejs]: done

**Pull request merged**